hadoop-hdds/dev-support/checkstyle/checkstyle.xml

```diff
@@ -73,6 +73,7 @@
 
     <module name="LineLength">
         <property name="fileExtensions" value="java"/>
+        <property name="max" value="120"/>
```

Comment:
this change is vetoed long back:
https://github.com/apache/ozone/pull/921#issuecomment-638231484

Reply:
@ayushtkn , thanks for pointing out [comment-Jun 15, 2020](https://github.com/apache/ozone/pull/921#issuecomment-638231484).

Indeed, that comment did not veto it.  The veto comment was [comment-Jun 14, 2020](https://github.com/apache/ozone/pull/921#issuecomment-643829659), which was based on
- https://accessibility.digital.gov/visual-design/typography/ 

It seems irrelevant since it talks about **textual information** but not computer code.  It also says
> ... Don’t make lines too long or too short: 45-75 characters per line ...

Are we going to enforce min line length 45?

Reply:
I am a strong +1 on committing this. Its a constant frustration trying to read and write this needlessly wrapped code.

Reply:
I personally don't have any issue with this change, a lot of projects have 120 as the limit(ex. hive)
But from the process point of view it isn't allowed by Apache ByLaws
https://www.apache.org/foundation/voting.html#votes-on-code-modification

It reads
``
but -1 votes are vetos and kill the proposal dead until all vetoers withdraw their -1 votes.
``

Regarding the veto
https://www.apache.org/foundation/voting.html#Veto

``
A -1 vote by a qualified voter stops a code-modification proposal in its tracks. This constitutes a veto, and it cannot be overruled nor overridden by anyone. Vetoes stand until and unless the individual withdraws their veto.
``

So, by this unless @anuengineer doesn't withdraw his veto, his vote can not be overruled

Reply:
@ayushtkn reading the same section further:

> the voter must provide with the veto a technical justification showing why the change is bad.  A veto without a justification is invalid and has no weight.

The soundness of the justification provided previously has been questioned.

Reply:
Just speaking about the process:
If you read the hadoop by laws as well
https://hadoop.apache.org/bylaws.html

> The validity of a veto, if challenged, can be confirmed by anyone who has a binding vote. This does not necessarily signify agreement with the veto - merely that the veto is valid.

Means that veto is valid only, Anu is a committer & he gave a reason, we are not convinced that doesn't make that invalid. This my understanding, & I have never seen a veto getting overruled, I maybe missing some catch there. Maybe we can relaunch that existing mail thread & someone with more insights on the process can help

Reply:
Well I don't want to be the bad guy blocking you folks for something I don't consider a problem(code wise).

But this is gonna set an example, so for posterity when this is used as an example by some other people(or me) to say a veto can be overruled by 1 or 2 people(coincidently from same org) on a PR itself in Ozone by stating the reason is invalid, which wasn't actually of 1+1=2 type, it was an accepted activity then, but I was in no way in favour of that. The other people didn't respect the ByLaws.

Reply:
> overruled by 1 or 2 people(coincidently from same org) 

@ayushtkn If you look at the previous PR, it was started by someone from other "org", and supported by way more than 1 or 2 people, also from various orgs, so please don't distort the facts

Reply:
>  to say a veto can be overruled by 1 or 2 people(coincidently from same org) 

@adoroszlai With all due respect none of them there said we tend to say @anuengineer vote isn't valid. The veto isn't valid is being said here. maybe my communication issues. I am talking about veto being overruled, not about the code change. If it was a normal proposal, I would have myself +1 this. 

Neverthless, I am nobody to stop anyone from clicking on the merge button neither I am aware what happens in this case, just lets remember & get this marked in the next Board Report for sure.

Reply:
@ayushtkn , I understand you concern -- I did mention explicitly in https://github.com/apache/ozone/pull/921#issuecomment-1841610769 that the veto should be considered as invalid.  The veto is actually similar to 1+1=2 type -- 45-75 line length is good for `textual information` but it definitely is not good for computer programs.  It is nonsense to enforce min line length in computer programs.  It is also nonsense to enforce 75 max line length.  How could it be valid?

On the other hands, short line length such as 80 does lead to many problems -- code readability, encouraging short variable names, unnecessary code formatting efforts, etc.  All these problems were not considered in the `textual information` recommandation.

Thanks for bring it up!